models/Asset.php

```diff
@@ -596,7 +596,7 @@ public function save()
             // refresh the inherited properties and update dependencies of each children
             if ($differentOldPath && isset($updatedChildren) && is_array($updatedChildren)) {
                 foreach ($updatedChildren as $updatedAsset) {
-                    $updatedAsset = self::getById($updatedAsset['id'], true);
+                    $updatedAsset = Asset::getById($updatedAsset, true);
```

Comment:
```suggestion
                    $updatedAsset = Asset::getById($updatedAsset, ['force' => true]);
```

Reply:
ahh right, thank you!
I've just forgot that we deprecated it on 10.5 and was somehow convinced that it was introduced in/since 10.6 🤦🏻